django/contrib/messages/storage/cookie.py

```diff
@@ -51,6 +47,14 @@ def decode(self, s, **kwargs):
         return self.process_messages(decoded)
 
 
+class MessageSerializer:
+    def dumps(self, obj):
+        return json.dumps(obj, separators=(',', ':'), cls=MessageEncoder).encode('latin-1')
+
+    def loads(self, data):
+        return json.loads(data.decode('latin-1'), cls=MessageDecoder)
+
+
```

Comment:
You've done well to limit the latin-1 charset to this class. :1st_place_medal:

Reply:
Note that this is due to the requirements in https://github.com/django/django/blob/28d998a41c92bdac8eeae6509ecc87ed044b30d0/django/core/signing.py#L81-L90

I am toying with the idea of adding an `encoder/decoder` class level variable to this class so I do not have to repeat the full class (even though it is really simple already)